src/task_manager/Robosub-State-Machine.xml

```diff
@@ -0,0 +1 @@
+<mxfile userAgent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36" version="8.0.8" editor="www.draw.io" type="device"><diagram name="Page-1" id="42789a77-a242-8287-6e28-9cd8cfd52e62">7Vrbkps4EP0aP8aFEOLyOOPMZFOV3UrVbO1mH2WQjTIyYoXwZb9+JRB3cFwZe5xk7Bej1g26T/dRN8zgYrP/IHAa/84jwma2Fe1n8P3MtoFj++pPSw6lxPPdUrAWNDKDGsET/Y8YoWWkOY1I1hkoOWeSpl1hyJOEhLIjw0LwXXfYirPurilek4HgKcRsKP2bRjI2UmBZTcdvhK5js7WPTMcSh89rwfPE7Dez4ar4ld0bXK1lxmcxjviuJYIPM7gQnMvyarNfEKZ1W6mtnPc40VvftyCJPGkCwRB4XmBFdogDH74DsFxii1lulPFMiynZjsow1qsLkmUkMg8gD5XSSKR0aJoJT9TffaEJovcCqhXLDTOXDC8Ju6+VteCMi2ZaJrGQd9qOPdmjvhX43qraBjlItUkSVTNChrOMhn/GNCk7zDRQtlqTvhIpD6aNc8mViAsZ8zVPMPvEeWpmZVLwZ1LdpTLq0kcOsuqeCiR67Ion8hFvKNPY/4uICCfYiM1OwDbt1oJW8dNyda8t+coPSRgq+dCyxtgZz0VIpsxpnAeLNZETY4JyjDZfa2GDlw+Eb4gUBzVAEIYl3XZdBBtPW9fjGrSpCwO4U8HnDMCHpQ4jJJUaeXvtQkQSkb1p8Pn2ErruK4AvQsSPnIuCD/444Btg70lbmSZrJf1DUdzMdpl6jPuIbtXlWl8+5onyzTs14GNCNVIzkmRcw3NRjVb30ZowsoaCS74pd8maDQvc77DCuvpnZKsJdqEZL8aJUlXjEwo3tvWu8AyZzad27XsLY4pKNbJ3MZXkKcWFEXeKzbuu0vhQAfuKrXRXyDc0NB3f8qgegGu8jQC4i8DH4vdyYGNG14n2TgVgpdIjiN4SIcn+KBZNr1ORvjnkQNPctU4MnpHFrcMCqE4554UvGuA3xSVPW2uFo1vE/AkiJjwhYjrXipjOUQg1hnhopPdhLrY1oL7mm9To1O/iSylJHL60G/9o3Mx7SOEp6ULEutQZ7ughrKf6y5g5uBYPDhOAwnf1/caaeP7NMatjinqG7PlGL5egF69HL3Vy3OYXOMIvgXMJenEHsFAZOhUFv6gjCdNgwGFMyfaNp4Y/C9c4JwQhdC2uQa/NNeDXZJpTjHw1phlm+1X6kqU4qZKXFBd5zgaL5yIZiogkoaQ8aeU67fE3NroIG1m9ZKeq6LbYyENDMkIXyXW8AXLiIv1WZkoZuSU7FyKgY8f085ITOiFuAXBudjJTP3OayAb6CII5DHykXMDxUQA7foACZ676HBs4HnRQhcxq9fIBzII9xNd3+H1OMMz3x6LnMueHohgUkk7sLIpWpdOUx/oVFZludSbcAuzVAiyCwwALnNFy0iXO+8EAXeY1UImWLF8WEBpUM2UuFFAsvqqHyljkWVmzL0cvxZFq6PSg8ULqrdB5eWjWVcwqEx2rdIIR8vfclyPTfYVMwJo7bjcbmFsQVoLPRNDirZPBwC+YJoBTXtVcLU8Aw1czJ9NaRkKeROO81o4xt7hx/rgB3R6luWAYN9BI3ABW8HIY+VMM9s0PGdTjya6VuwYyVmtbwogGWtTKoiFmd6ZjQ6OITcGqi6UJbY/YZPoNleN3A7czVkIcMYB9hqxt8gTxlvTfT5pfUf9VufgtG6BWgjGAE7ymAYa0dfuU6kf4lKp17nNR0Dv32c7kuW9P5ZfWdVP7ONMp7JSix0sPYRMlDs+eO+oMjILA9i1YcXLlNQDNVexSNwJtz4HI6xU5yocaFDkGu0DPmtvABa4PfWB5dq+SAtEcuYq3gYcsFHiwu8l3V1JUs/meshzefLQKH/4H</diagram></mxfile>
```

Comment:
Can this be stored as a PNG instead of XML?

Reply:
You included the png so you can remove this file

Reply:
If you save as in draw.io as *.png it automatically embeds the diagram data and lets you open the file from a png again. Try it out and let me know if it works